Студентам для решения домашнего задания/case_1.py

```diff
@@ -0,0 +1,59 @@
+import os, glob, csv, re, chardet
+
+path = 'C:\\Users\\Администратор\\Desktop\\Студентам для решения домашнего задания'
+file_for_write = 'C:\\Users\\Администратор\\Desktop\\Студентам для решения домашнего задания\\main_data.txt'
+
+def get_data(path):
+    files = []
+    os_prod_list = []
+    os_name_list = []
+    os_code_list = []
+    os_type_list = []
+    main_data = []
+
+    for filename in glob.glob(os.path.join(path, '*.txt')):
+        if re.match(r'main_data',file_for_write):
+            continue
+        else:
+            files.append(filename)
+            
+            with open(filename) as f_n:
+                f_n_reader = csv.reader(f_n)

```

Comment:
Opening a file without specifying an encoding is not allowed.
But first of all, this encoding must be determined by any known method.